lib/tasks/vacancies_to_update.yaml

```diff
@@ -212,3 +212,477 @@ We can offer:
+      DURHAM MARTYRS MULTI ACADEMY TRUST</p>"
+      subject: "English"
+    -
+      slug: 'curriculum-leader-of-mathematics'
+      job_title: "Curriculum Leader of Mathematics"
+      job_description: "
+      <p>For September 2018, we are looking for a dynamic and enthusiastic leader who is able to engage with and inspire others in order to improve progress of students in the mathematics department. This is a full time, permanent appointment.</p>
+      <p>Applicants will be highly motivated and have a passion to ensure that our learners achieve the very best that they can at all times.  The core purpose of the role will be to uphold the fundamental standards of pride, respect and achievement with all learners but also ensure that all students receive the highest standard of bespoke support.</p>
+      <p>The college is focused upon rapidly raising standards of achievement and this makes it an exciting place to work. We’re committed to ensuring that every student reaches their potential in a calm and engaging environment. We are seeking a driven individual who can lead the mathematics department to provide the highest quality teaching and learning, with the best possible student support and the most appropriate and innovative curriculum.</p>
+      <p>The successful candidate will put teaching and learning first and be committed to providing a high quality and inclusive learning experience for all.  Furthermore, they must be committed to the professional development of all mathematics staff. The department comprises 7 full time teachers and 1 part-time member of staff.</p>
+      <p>All post information can be downloaded either from this advert or from http://www.burnsidecollege.org.uk/vacancies/curriculum-leader-of-mathematics-mps-ups-tlr-1c/</p>
+      <p>If you would like any further information, or to discuss this role in more detail, please contact Daniel Jamieson, Headteacher.</p>
+      <p>In addition to the application form please provide a covering letter, of no more than two sides of A4, in which you outline your suitability for the post as well as your philosophy on teaching.  Completed applications should be returned to Mrs L Heide, Burnside Business and Enterprise College, St. Peter’s Road, Wallsend, NE28 7LQ or electronically to l.heide@burnsidecollege.org.uk</p>
+      <p>Telephone: 0191 2598500 </p>
+      <p>Burnside is committed to safeguarding and all offers of employment are subject to satisfactory enhanced DBS clearance.</p>
+      <p>Closing date:  9am Monday 14th May 2018</p>
+      <p>Interview date: Thursday 17th May 2018</p>"
+      experience: "The successful candidate will have a strong track record in leadership, and a passion for teaching and learning. In addition, you will have experience of delivering consistently excellent mathematics teaching across key stages"
+    -
+      slug: 'teacher-of-history-maternity-cover'
+      job_title: 'Teacher of History (part time maternity cover)'
```

Comment:
1) Delete duplicate copy of this listing
2) Pay scale range should be Main pay range 1 to Upper pay range 3